src/stash/src/upgrade.rs

```diff
@@ -58,6 +60,20 @@ pub(crate) mod v33_to_v34;
 pub(crate) mod v34_to_v35;
 pub(crate) mod v35_to_v36;
 
+macro_rules! objects {
+    ( $( $x:ident ),* ) => {
+        $(
+            paste! {
+                pub mod [<objects_ $x>] {
```

Comment:
While we're here, can you make this non-pub, or at least `pub(crate)`? I just want to make sure they're only referenced from upgrades